src/AccessibilityInsights.CommonUxComponents/Controls/CustomControlOverridingAutomationPeer.cs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System.Windows.Automation.Peers;
+using System.Windows.Controls;
+
+namespace AccessibilityInsights.CommonUxComponents.Controls
+{
+    /// <summary>
+    /// CustomControlOverridingAutomationPeer class
+    /// class to override LocalizedControlType and control/content element values for custom control(UserControl)
+    /// </summary>
+    public class CustomControlOverridingAutomationPeer: UserControlAutomationPeer
+    {
+        /// <summary>
+        /// Value for custom IsControlElement
+        /// </summary>
```

Comment:
Comment doesn't match code. I personally prefer the full name over the abbreviated form, but that's probably not a change you're trying to introduce here

Reply:
Will address this and other code-related comments in a future PR that removes the existing controls from SharedUx.